.storybook/preview.jsx

```diff
@@ -15,7 +15,6 @@ export default {
         )
     ],
     parameters: {
-        layout: "centered",
```

Comment:
@darkwebdev this one fixes the narrow issue. By default storybook uses `padding` so it has some padding around.